### PR TITLE
DOC Add details about ModelAdmin and Login Forms to 4.6 changelogs

### PR DESCRIPTION
**Pull request by maxime-rainville**
After some reflection I concluded that the ModelAdmin API change and the login form change are worth highlighting in the 4.6.0 changelog.

### CODE REVIEW

File: docs/en/04_Changelogs/4.6.0.md [hunk 1]
```diff
@@ -75,7 +77,29 @@ module, you should upgrade it as well.
...
+* `updateGridFieldConfig()`.
+
+This will make it easier for developers to customise GridFields in their ModelAdmins.
+
+Learn how to [Alter the ModelAdmin GridField or Form](/developer_guides/customising_the_admin_interface/modeladmin/#altering-the-modeladmin-gridfield-or-form)
+
+## Login forms module ships with installer {#loginforms}
+
+The `silverstripe/login-forms` module is now part of the default installer. This alters the login form to provide
+consistent styling and behaviour that's independent from the specifics in your project. Only new project will
```
Comment (michalkleiner) [inline docs/en/04_Changelogs/4.6.0.md:97]: 'Only a new project' or 'Only new projects'

Comment (maxime-rainville) [inline docs/en/04_Changelogs/4.6.0.md:97]:
```suggestion
consistent styling and behaviour that's independent from the specifics in your project. Only new projects will
```

Comment (emteknetnz) [reply docs/en/04_Changelogs/4.6.0.md:97]: @maxime-rainville could you commit this suggestion (I'm unable to do this myself), I'll merge once you've done this

Comment (maxime-rainville) [reply docs/en/04_Changelogs/4.6.0.md]: Humm ... I would have assumed you should be able to do it. Is the "commit suggestion" button disabled for you?

File: docs/en/04_Changelogs/4.6.0.md [hunk 2]
```diff
@@ -75,7 +77,29 @@ module, you should upgrade it as well.
...
+* `getGridField()`
+* `getGridFieldConfig()`.
+
+Two matching extension hooks have been added as well:
+* `updateGridField()`
+* `updateGridFieldConfig()`.
+
+This will make it easier for developers to customise GridFields in their ModelAdmins.
+
+Learn how to [Alter the ModelAdmin GridField or Form](/developer_guides/customising_the_admin_interface/modeladmin/#altering-the-modeladmin-gridfield-or-form)
```
Comment (emteknetnz) [inline docs/en/04_Changelogs/4.6.0.md:92]:
```suggestion
Learn how to [alter the ModelAdmin GridField or Form](/developer_guides/customising_the_admin_interface/modeladmin/#altering-the-modeladmin-gridfield-or-form)
```

**Comment by maxime-rainville** [comment]
I've rebased this PR and fixed the conflict.

**Pull request merged**